source/about/desktop-app-changelog.md

```diff
@@ -12,7 +12,14 @@ This changelog summarizes updates to Mattermost desktop app releases for [Matter
 (release-v5-11)=
 ## Release v5.11 (Extended Support Release)
 
-**Release day: February 14th, 2025**
+- **v5.11.1, released **
```

Comment:
Question: Given that v5.11 introduces a breaking change for customers running v9.3 or earlier server releases, does it make sense to over-communicate that breaking change across all dot releases for v5.11?

Reply:
A note will be added after https://github.com/mattermost/desktop/pull/3348 is included in the dot release.